charts/templates/NOTES.txt

```diff
@@ -0,0 +1,16 @@
+
+Successfully installed OpenEBS.
+
+Check the status by running: kubectl get pods -n {{ .Release.Namespace }}
+
+The default values will install both Local and Replicated Engines. However,
+the Replicated Engine will require additional configuration to be fuctional.
+The Local Engine offers non-replicated local storage using 3 different storage
+backends, while the Replicated Engine provides replicated highly-available
```

Comment:
```suggestion
backends, while the Replicated one provides replicated highly-available
```

Reply:
Please re review the content, have tried to change is as per last naming discussion.